packages/salesforcedx-vscode-apex/src/commands/forceAnonApexExecute.ts

```diff
@@ -168,6 +166,7 @@ export class AnonApexLibraryExecuteExecutor extends LibraryCommandletExecutor<
       return false;
     }
 
+    fs.mkdirSync(path.dirname(logFilePath), { recursive: true });
```

Comment:
Ensure the `.sfdx/tools/debug/logs` dir is present

Reply:
So this line is the fix for https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001V4nITYAZ/view, right?  This line creates the `.sfdx/tools/debug/logs` directory if it doesn't already exist in the project?